lms/routes.py

```diff
@@ -5,6 +5,8 @@ def includeme(config):
     config.add_route("assets", "/assets/*subpath")
     config.add_route("status", "/_status")
     config.add_route("favicon", "/favicon.ico")
+    config.add_route("ui-playground-root", "/ui-playground")
+    config.add_route("ui-playground", "/ui-playground/*subpath")
```

Comment:
Our UI has client side routes so I wanted to capture them all, is this the right way?
e.g
http://localhost:8001/ui-playground/foo
http://localhost:8001/ui-playground/bar

Reply:
I think just `/ui-playground/{subpath}` would work but only if `subpath` does not contain any `/`'s

Reply:
Using a `*remainder` pattern is the right approach here.